apps/jobs/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.conf.urls import url
-# from .views import ()
 
+from . import views
 
 urlpatterns = [
+    url(r'challenge/(?P<challange_id>[0-9]+)/'
+        r'challenge_phase/(?P<challange_phase_id>[0-9]+)/',
```

Comment:
So, since it is in the `jobs` app, so the url would be `/api/jobs/challenge/<challenge_id>/challenge_phase<phase_id>/submission`. Does that sound good?